src/packages/verticaltabs/doc.zh-TW.md

```
@@ -0,0 +1,135 @@
+
+| 屬性 | 說明 | 類型 | 默認值 |
+| --- | --- | --- | --- |
+| title | 標題 | `string` | `-` |
+| value | 標簽 Key , 匹配的標識符, 默認為索引值 | `string` \| `number` | `-` |
+| disabled | 是否禁用標簽 | `boolean` | `false` |
+
+## 主題定製
+
+### 樣式變量
+
+組件提供了下列 CSS 變量，可用於自定義樣式，使用方法請參考 [ConfigProvider 組件](#/zh-CN/component/configprovider)。
```

Markdownlint

Link fragments should be valid

(MD051, link-fragments)